src/configmanager.h

```diff
@@ -137,12 +137,17 @@ class ConfigManager
 			LAST_INTEGER_CONFIG /* this must be the last one */
 		};
 
+		enum floating_config_t {
+			LAST_FLOATING_CONFIG
```

Comment:
arrays cant be zero sized, you have to mark this 1 or add at least one config enum